src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,7 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
```

Comment:
@le717 The problem is in the test in this line of code in `testConvert()`:

```js
    expect(colorEditor.getColor().toString()).toBe(result);
```

`colorEditor.getColor()` returns a tinycolor with `_originalInput` correctly stored as UPPER case. But tinycolor `toString()` seems to ignore `_originalInput`, so I think this is a bug in tinycolor.